rover_msgs/GPSData.lcm

```diff
@@ -1,14 +1,51 @@
 package rover_msgs;
 
 struct GPSData {
+	//UTC time hhmmss.sss format
+	//Will be "0" if empty
 	double timeStamp;
-	double latitude;
-	byte latitudeDirection;
-	double longitude;
-	byte longitudeDirection;
+	
+	//Degrees
```

Comment:
I like these comments indicating units and default values. Make them inline comments so its a bit easier to parse the file and identify the struct members rather than having to pick them out from between comments